### PR TITLE
Hardware: Add support for Pixhawk FMUv6XRT

### PR DESCRIPTION
**Pull request by mrpollo**
This adds support for the new Pixhawk FMUv6XRT Open Standard based on NXP Semiconductor MCU, please note this is using the Dronecode vendorID

cc @davids5 @dk7xe

### CODE REVIEW

File: src/comm/USBBoardInfo.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
         { "vendorID": 7052, "productID": 54,        "boardClass": "Pixhawk",    "name": "PX4 FMU V6U" },
         { "vendorID": 12677, "productID": 53,       "boardClass": "Pixhawk",    "name": "PX4 FMU V6X" },
         { "vendorID": 12677, "productID": 56,       "boardClass": "Pixhawk",    "name": "PX4 FMU V6C" },
+        { "vendorID": 3643, "productID": 29,        "boardClass": "Pixhawk",    "name": "PX4 FMU V6X-RT" },
```
Comment (davids5) [inline src/comm/USBBoardInfo.json:20]: @mrpollo I think theses are all in decimal.

The number in the email were in HEX.  Both need to be converted to decimal.

**Review by davids5 — CHANGES_REQUESTED** [review]
See inline

File: src/comm/USBBoardInfo.json (hunk 1 above)
Comment (mrpollo) [reply src/comm/USBBoardInfo.json]: Good catch! Thanks @davids5

**Review by davids5 — APPROVED** [review]
LGTM

**Comment by dk7xe** [comment]
Thank you @mrpollo and @davids5 !

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**